app/models/user.rb

```diff
@@ -10,7 +51,9 @@ class User < ApplicationRecord
   has_many :org_edits, through: :permissions, source: :organization
   has_many :prog_edits, through: :permissions, source: :program
 
+  accepts_nested_attributes_for :media, allow_destroy: true
```

Comment:
Why do we need to accept nested attributes for media? Is this leftover from when we were trying to get a user to save with pre-built media in their collection or do you still need it for the transaction that you made?

Reply:
It's leftover, I'll delete it